src/meta/meta_bulk_load_service.cpp

```diff
@@ -1661,5 +1661,48 @@ void bulk_load_service::create_missing_partition_dir(const std::string &app_name
         });
 }
 
+// ThreadPool: THREAD_POOL_META_SERVER
+void bulk_load_service::check_app_bulk_load_states(std::shared_ptr<app_state> app,
+                                                   bool is_app_bulk_loading)
+{
+    std::string app_path = get_app_bulk_load_path(app->app_id);
+    _meta_svc->get_remote_storage()->node_exist(
+        app_path, LPC_META_CALLBACK, [this, app_path, app, is_app_bulk_loading](error_code err) {
+            if (err == ERR_TIMEOUT) {
+                ddebug_f(
+                    "check app({}) bulk load dir({}) timeout, try later", app->app_name, app_path);
+                tasking::enqueue(LPC_META_CALLBACK,
+                                 nullptr,
+                                 std::bind(&bulk_load_service::check_app_bulk_load_states,
+                                           this,
+                                           app,
+                                           is_app_bulk_loading),
+                                 0,
+                                 std::chrono::seconds(1));
+                return;
+            } else {
```

Comment:
Now that you have returned in the previous condition, you can remove `else`, it will be more friendly to read with less indents.